### PR TITLE
fix(pytest): Add master restart delay

### PR DESCRIPTION
**Pull request by dranikpg**
Add small delay before re-starting master

### CODE REVIEW

File: tests/dragonfly/__init__.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ def stop(self, kill=False):
         except subprocess.TimeoutExpired:
             print("Unable to terminate DragonflyDB gracefully, it was killed")
             proc.kill()
+            proc.communicate()
```
Comment (romange) [inline tests/dragonfly/__init__.py:55]: why not print the results? https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate
or does it print it internally?

**Comment by romange** [comment]
Should we also increase the test timeout?
unrelated, maybe increase the timeout from 30min to 45?

File: tests/dragonfly/__init__.py (hunk 1 above)
Comment (adiholden) [reply tests/dragonfly/__init__.py:55]: Did you see the print above when test run? and where did you see the port print?
I found this fail for the disconnect replication fail and I dont see those prints
https://github.com/dragonflydb/dragonfly/actions/runs/4335282784/jobs/7569761257#step:6:1582

Comment (dranikpg) [reply tests/dragonfly/__init__.py:55]: Oh, its unrelated sorry, I just added this line for correctness

**Review by romange — APPROVED** [review]

**Pull request merged**